lib/WP_Auth0_LoginManager.php

```diff
@@ -245,30 +244,29 @@ public function redirect_login() {
 
     // Attempt to authenticate with the Management API
     $client_credentials_token = WP_Auth0_Api_Client::get_client_token();
-    $userinfo_resp = null;
+    $userinfo_resp_code = $userinfo_resp_body = null;
 
     if ( $client_credentials_token ) {
-
-        $userinfo_resp = WP_Auth0_Api_Client::get_user(
-            $this->a0_options->get( 'domain' ),
-            $client_credentials_token,
-            $decoded_token->sub
-        );
+      $userinfo_resp = WP_Auth0_Api_Client::get_user( $domain, $client_credentials_token, $decoded_token->sub );
+      $userinfo_resp_code = (int) wp_remote_retrieve_response_code( $userinfo_resp );
+      $userinfo_resp_body = wp_remote_retrieve_body( $userinfo_resp );
     }
 
-    $userinfo_resp_code = (int) wp_remote_retrieve_response_code( $userinfo_resp );
-    $userinfo_resp_body = wp_remote_retrieve_body( $userinfo_resp );
-
-    // Management API call failed
+    // Management API call failed, fallback to userinfo
```

Comment:
So why does Management API fail?

Reply:
This is something we should have had in 3.5.0. It will fail if the Client/Application is not authorized on the Management API or if Client Grant has not been turned on. Has been a common support theme and wanted to get it in there for anyone else upgrading.